src/components/CldPoster.vue

```diff
@@ -11,9 +11,9 @@ export default {
   inheritAttrs: false,
 
   render() {
-    console.error(
-      `<cld-poster /> should be a immediate descendant of cld-video`
-    );
+    // console.error(
+    //   `<cld-poster /> should be a immediate descendant of cld-video`
+    // );
```

Comment:
@mayashavin , please remove comments

Reply:
@asisayag2 Removed it in the next PR for CLDVideo and CLDPoster :)